schemas.json

```diff
@@ -191,15 +191,24 @@
     },
     "additionalProperties": false
   }
-"game_score": {
-    "definition": "The patient score in the game",
+"left_hand_score": {
```

Comment:
Note for the future, we won't have to update this file anymore, the updated schemas will be stored in intit_engine_db.sql